### PR TITLE
scan: implement ibc msg ui and wireup

### LINKED ISSUE #1210: Implement IBC Msg UI and Wire up
- Implement IBC Msg and use it on Latest Tx Table on FrontPage and All Tx Page
- Use TxSub

### PR DESCRIPTION
**Pull request by tansawit**
Fixes: #1210

### CODE REVIEW

File: scan/src/pages/RequestIndexPage.re [hunk 1]
```diff
@@ -50,6 +50,8 @@ let make = (~reqID) =>
     let%Sub request = requestSub;
     let%Sub blockCount = blockCountSub;
 
+    Js.Console.log(request.id);
```
Comment (evilpeach) [inline scan/src/pages/RequestIndexPage.re]: remove log

**Review by evilpeach — CHANGES_REQUESTED** [review]

File: scan/src/subscriptions/TxSub.re [hunk 2]
```diff
@@ -227,6 +227,308 @@ module Msg = {
...
+      chainID: string,
+      trustingPeriod: MomentRe.Duration.t,
+      unbondingPeriod: MomentRe.Duration.t,
+    };
+    let decode = json => {
+      JsonUtils.Decode.{
+        address: json |> field("address", string) |> Address.fromBech32,
+        clientID: json |> field("client_id", string),
+        chainID: "band-consumer",
+        // TODO: Change to use MomentRe
```
Comment (evilpeach) [inline scan/src/subscriptions/TxSub.re]: remove TODO

File: scan/src/subscriptions/TxSub.re [hunk 3]
```diff
@@ -227,6 +227,308 @@ module Msg = {
...
+
+  module Packet = {
+    type t = {
+      sender: Address.t,
+      sequence: float,
+      sourcePort: string,
+      sourceChannel: string,
+      destinationPort: string,
+      destinationChannel: string,
+      timeoutHeight: float,
```
Comment (evilpeach) [inline scan/src/subscriptions/TxSub.re]: int

File: scan/src/subscriptions/TxSub.re [hunk 4]
```diff
@@ -227,6 +227,308 @@ module Msg = {
...
+    };
+    let decode = json => {
+      JsonUtils.Decode.{
+        sender: json |> field("signer", string) |> Address.fromBech32,
+        sequence: json |> at(["packet", "sequence"], float),
+        sourcePort: json |> at(["packet", "source_port"], string),
+        sourceChannel: json |> at(["packet", "source_channel"], string),
+        destinationPort: json |> at(["packet", "destination_port"], string),
+        destinationChannel: json |> at(["packet", "destination_channel"], string),
+        timeoutHeight: json |> at(["packet", "timeout_height"], float),
```
Comment (evilpeach) [inline scan/src/subscriptions/TxSub.re]: int

File: scan/src/subscriptions/TxSub.re [hunk 5]
```diff
@@ -227,6 +227,308 @@ module Msg = {
...
+
+  module Acknowledgement = {
+    type t = {
+      sender: Address.t,
+      sequence: int,
+      sourcePort: string,
+      sourceChannel: string,
+      destinationPort: string,
+      destinationChannel: string,
+      timeoutHeight: string,
```
Comment (evilpeach) [inline scan/src/subscriptions/TxSub.re]: int

File: scan/src/subscriptions/TxSub.re [hunk 6]
```diff
@@ -227,6 +227,308 @@ module Msg = {
...
+
+  module Timeout = {
+    type t = {
+      sender: Address.t,
+      sequence: float,
+      sourcePort: string,
+      sourceChannel: string,
+      destinationPort: string,
+      destinationChannel: string,
+      timeoutHeight: float,
```
Comment (evilpeach) [inline scan/src/subscriptions/TxSub.re]: int

File: scan/src/subscriptions/TxSub.re [hunk 7]
```diff
@@ -227,6 +227,307 @@ module Msg = {
...
+      };
+  };
+
+  module ConnectionOpenTry = {
+    type t = {
+      signer: Address.t,
+      chainID: string,
+      connectionID: string,
+      clientID: string,
+      consensusHeight: ID.Block.t,
```
Comment (evilpeach) [inline scan/src/subscriptions/TxSub.re]: int

**Review by evilpeach — CHANGES_REQUESTED** [review]

**Review by evilpeach — APPROVED** [review]

**Review by prin-r — APPROVED** [review]

**Pull request merged**